cmd/root.go

```diff
@@ -17,7 +18,7 @@ import (
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
-	Use:   "valida --test spec.json",
+	Use:   "valida --test",
```

Comment:
Ini better tetep dikasih hint ham, paling gini aja

```suggestion
	Use:   "valida --test [JSON FILE]",
```

Reply:
Done lang